addons/mk6mortar/functions/fnc_mortarInit.sqf

```diff
@@ -18,27 +18,28 @@
 params ["_mortar"];
 
 if (_mortar getVariable [QGVAR(initialized),false] || _mortar getVariable [QGVAR(exclude),false]) exitWith {TRACE_1("Exit",_mortar)};
+if (!(_mortar turretLocal [0])) exitWith {TRACE_1("Exit - turret not local",_mortar)};
 
-// Remove all magazines
+// Remove all magazines from turret
 if (count magazines _mortar > 0) then {
     {
-        [QGVAR(removeMagazine), [_mortar, _x]] call EFUNC(common,globalEvent);
+        _mortar removeMagazineTurret [_x,[0]];
     } forEach magazines _mortar;
 };
 
-// Replace current weapon with ammo handling weapon
+// Replace current turret weapon with ammo handling weapon
 private _currentWeapon = _mortar weaponsTurret [0] select 0;
 private _newWeapon = "";
 
-if (_currentWeapon == "mortar_82mm") then {
+if (tolower _currentWeapon == "mortar_82mm") then {
```

Comment:
technically not needed, because `==` compares strings case insensitive, but whatever